src/components/button/CategoryButton.style.tsx

```diff
@@ -1,14 +1,34 @@
-import { Button, styled } from '@mui/material';
+import { Button, ButtonProps, styled } from '@mui/material';
 
-export const StyledOnCategoryButton = styled(Button)({
-  borderRadius: '4px',
-  background: '#203C8E',
-  color: '#FFFFFF',
-});
+interface Props {
+  active?: boolean;
+}
+
+const CategoryDefaultStyleButton = (props: ButtonProps) => {
+  return (
+    <Button
+      sx={{
+        display: 'flex',
+        width: '51px',
+        padding: '12px 4px',
+        flexDirection: 'column',
+        alignItems: 'center',
+        gap: '8px',
+      }}
+      variant={'contained'}
+      {...props}
+    >
+      {props.children}
+    </Button>
+  );
+};
```

Comment:
styled를 사용하면 아래처럼 심플하게 수정 가능합니다.

```suggestion
const CategoryDefaultStyleButton = styled(Button)`
  display: flex;
  width: 51px;
  padding: 12px 4px;
  flex-direction: column;
  align-items: center;
  gap: 8px;
`;
```